system/include/emscripten/em_asm.h

```diff
@@ -167,6 +167,8 @@ __attribute__((nothrow))
 void emscripten_asm_const_async_on_main_thread(
   const char* code, const char* arg_sigs, ...);
 
+void emscripten_unwrap_js_handle(int js_handle);
```

Comment:
nit: "unwrap" makes it sound like it's returning something

Reply:
Renamed to "release"